adapters/sidekiq/lib/opentelemetry/adapters/sidekiq/middlewares/client/tracer_middleware.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Sidekiq
+      module Middlewares
+        module Client
+          class TracerMiddleware
+            def call(worker_class, job, _queue, _redis_pool)
+              tracer.in_span(
+                worker_class,
+                attributes: job,
```

Comment:
This is fine for now, but once we open this up to more 👀 we should discuss this. In our instrumentation, we strip job arguments: https://github.com/Shopify/shopify-tracing/blob/816ba2aef3c6ee8a232766028181b7b1ca03a2b1/lib/shopify/tracing/adapters/sidekiq_adapter.rb#L18-L19

We either need to allow configurable filtering, or default to a minimal set of attributes and encourage users to add things that are meaningful to them. In my experience, adding the full set of job arguments is a bad idea (e.g. we've seen jobs with PII in their args, jobs with multi-megabyte args, and generally a lot of noise).

Reply:
I'm not convinced it makes sense to include the job args as attributes for both the producer and consumer spans. They make sense to me for the consumer side, but the producer likely doesn't need to be so verbose.

Reply:
>we've seen jobs with PII in their args
Never considered people would be doing that... yikes

>jobs with multi-megabyte args
That's impressive...

With those points in mind, I'll switch this to *only* include the `jid`  and `created_at` fields to start.